scripts/send-message.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I used this a bit when testing locally. Might be worth keeping around, but if you guys don't find it useful I'm okay removing it too

Reply:
Looks useful, but maybe add some docs what setup is expected for this to work (I presume it simply running the test network locally, right?)